frontend/src/scenes/insights/ComputationTimeWithRefresh.tsx

```diff
@@ -5,23 +5,27 @@ import { usePeriodicRerender } from 'lib/hooks/usePeriodicRerender'
 import { LemonButton } from '@posthog/lemon-ui'
 
 export function ComputationTimeWithRefresh(): JSX.Element | null {
-    const { lastRefresh, insightRefreshButtonDisabledReason } = useValues(insightLogic)
```

Comment:
We need to make analogous changes in `frontend/src/queries/nodes/InsightViz/ComputationTimeWithRefresh.tsx`. The difference is that the `frontend/src/scenes/insights/` code uses `insightLogic`, while `frontend/src/queries/nodes/InsightViz/` uses the new `dataNodeLogic`.